composer.json

```diff
@@ -14,14 +14,14 @@
     "require": {
         "php": ">=7.2",
         "neitanod/forceutf8": "^2.0.4",
-        "php-ffmpeg/php-ffmpeg": "^0.14.0"
+        "php-ffmpeg/php-ffmpeg": "dev-master"
```

Comment:
Why `dev-master` and not `^0.17.0` ? 
It seems that https://github.com/PHP-FFMpeg/PHP-FFMpeg/releases/tag/v0.17.0 add support for php8. I fear that "dev-master" would put us at risk to have a broken release at some point.